core-lib/Benchmarks/SavinaCSP.ns

```diff
@@ -60,4 +60,217 @@ class SavinaCSP usingPlatform: platform andHarness: harness = Value (
     public newInstance = ( ^ self new: 2000 )
     public setupVerifiedRun: run = ( run problemSize )
   )
+
+
+    public class ForkJoinThroughput new: numProcesses  totalMessages: numMessages = Benchmark <: Value (
```

Comment:
Should we add the CSP benchmarks to our usual benchmark runs/rebench setup?

I think, now that you are doing research on the CSP stuff, it would be good to make sure things don't break.

Reply:
good idea, should i add a new savinaCSP suite to the codespeed.conf

Reply:
yes, that would be good. Thanks

Reply:
added codespeed conf entries and a gitlab ci job